scripts/xnat/sibis.py

```diff
@@ -7,4 +7,8 @@ def logging(id, message, **kwargs):
     log = dict(experiment_site_id=id,
                error=message)
     log.update(kwargs)
-    print(json.dumps(log, sort_keys=True))
+    try:
+        print(json.dumps(log, sort_keys=True))
+    except Exception as e:
+        print("ERROR: Failed to serialize to JSON: %s" % e)
+        print("Original log message: %s" % log)
```

Comment:
same thing here, best to figure out what it is trying to serialize to json and possibly cast it as a string.  Also, this sibis.py file is copied in each dir as a temporary fix for a library  that will replace it. See https://github.com/sibis-platform/sibis